apps/web/modules/core/rate-limit/helpers.ts

```diff
@@ -0,0 +1,33 @@
+import { hashString } from "@/lib/hashString";
```

Comment:
can we rename this file to hash-string.ts for similarity for other files and this also follows the new code guidelines 🙏